scope/utils.py

```diff
@@ -102,6 +102,60 @@ def plot_light_curve_data(
         plt.savefig(save)
 
 
+def plot_periods(
+    features: pd.DataFrame,
+    limits: Optional[list] = None,
+    loglimits: Optional[bool] = False,
+    number_of_bins: Optional[int] = 20,
+    title: Optional[str] = None,
+    save: Optional[str] = None,
+):
+    """Plot a histogram of periods for the sample
+
+    """
+    # plot the H-R diagram for 1 M stars within 200 pc from the Sun
+    plt.rc("text", usetex=True)
+
+    # make figure
+    fig, ax = plt.subplots(figsize=(6, 6))
+    if title is not None:
+        fig.suptitle(title, fontsize=24)
+
+    if limits is not None:
+        if loglimits:
+            edges = np.logspace(np.log10(limits[0]), np.log10(limits[1]),
```

Comment:
Lint here and below. How come the linter didn't complain?

Reply:
again, what is failing?